Sources/python/SimEx/Utilities/wpg_to_opmd.py

```diff
@@ -0,0 +1,88 @@
+import sys
+import numpy
+import math
+import h5py
+import os
+
+from SimEx.Utilities import OpenPMDTools as opmd
+
+def _convertToOPMD(h5_in):
+    """ Take native wpg output and rewrite in openpmd conformant way. """
+
+    # Load the generated h5 file.
+    #wpg_outputs = [os.path.join( self.output_path, f) for f in os.listdir( self.output_path ) ]
+    wpg_outputs = [h5_in]
+    wpg_outputs = [f for f in wpg_outputs if f.split('.')[-1] == 'h5']
+    print wpg_outputs
+
+    for i,f in enumerate(wpg_outputs):
+        h5 = h5py.File( f, 'r')
+        opmd_h5 = h5py.File(f+'.opmd', 'w')
+
+        # Get number of time slices in wpg output.
+        data_shape = h5['data/arrEhor'].value.shape
+        number_of_x_meshpoints = data_shape[0]
+        number_of_y_meshpoints = data_shape[1]
+        number_of_time_steps = data_shape[2]
+
+        for it in range(number_of_time_steps):
+            # Write opmd
+            # Setup the root attributes for iteration 0
+            opmd.setup_root_attr( opmd_h5 )
+
+            full_meshes_path = opmd.get_basePath(opmd_h5, it) + opmd_h5.attrs["meshesPath"]
+            # Setup basepath.
+            opmd.setup_base_path( opmd_h5, iteration=it)
+            opmd_h5.create_group(full_meshes_path)
+            meshes = opmd_h5[full_meshes_path]
+            # Path to the E field, within the h5py file
+            full_e_path_name = b"E"
+            meshes.create_group(full_e_path_name)
+            E = meshes[full_e_path_name]
+
+
+            # Create the dataset (2d cartesian grid)
+            E.create_dataset(b"x", (number_of_x_meshpoints, number_of_y_meshpoints), dtype=numpy.complex64)
```

Comment:
need to cross-check how this looks in C++

we did not define specific types for complex dates yet (but could stick with the h5py types).